libnavigation-base/src/main/res/values-fr/strings.xml

```diff
@@ -1,15 +1,15 @@
 <resources xmlns:tools="http://schemas.android.com/tools">
 
     <!-- Time formatting -->
-    <plurals name="mapbox_number_of_days">
-        <item quantity="one" tools:ignore="ImpliedQuantity">jour</item>
+    <plurals name="mapbox_number_of_days" tools:ignore="MissingTranslation">
+        <item quantity="one">jour</item>
         <item quantity="other">jours</item>
     </plurals>
     <!-- Hour -->
-    <string name="mapbox_unit_hr">h</string>
+    <string name="mapbox_unit_hr" tools:ignore="MissingTranslation">h</string>
     <!-- Minute -->
-    <string name="mapbox_unit_min">min</string>
+    <string name="mapbox_unit_min" tools:ignore="MissingTranslation">min</string>
 
     <!-- Notification Strings -->
     <string name="mapbox_end_navigation">Fin de navigation</string>
-</resources>
+    </resources>
```

Comment:
That is unnecessary. Or was it some script that moved the line?

Reply:
Yeah that's Transifex after `tx pull -a` 😥